pkgs/desktops/gnome-3/misc/gnome-tweaks/default.nix

```diff
@@ -23,7 +23,7 @@ in stdenv.mkDerivation rec {
     libnotify gnome3.gnome-shell python3Packages.pygobject3
     libsoup gnome3.gnome-settings-daemon gnome3.nautilus
     gnome3.mutter gnome3.gnome-desktop gobjectIntrospection
-    gnome3.nautilus
+    gnome3.nautilus gnome3.gnome-shell-extensions
```

Comment:
Could you add comment that it is needed for schemas from the theme extension?

Reply:
Ah, true